citesphere/src/main/java/edu/asu/diging/citesphere/core/service/impl/CitationManager.java

```diff
@@ -506,6 +510,27 @@ public CitationPage getPrevAndNextCitation(IUser user, String groupId, String co
     public void deleteLocalGroupCitations(String groupId) {
         citationStore.deleteCitationByGroupId(groupId);
     }
+    
+    @Override
+    public Citations findAuthorityItems(IAuthorityEntry entry, IUser user) {       
```

Comment:
that's a confusing name. Why "items"? and it's not finding "authority items", it's finding citations that refer to the given authority, doesn't it?

Reply:
also needs tests